web/src/recent_topics_ui.js

```diff
@@ -728,6 +728,22 @@ function topic_sort(a, b) {
     return sort_comparator(topic_sort_key(a), topic_sort_key(b));
 }
 
+function unread_count(conversation_data) {
+    const message = message_store.get(conversation_data.last_msg_id);
+    return message_to_conversation_unread_count(message);
+}
+
+function unread_sort(a, b) {
+    const a_unread_count = unread_count(a);
+    const b_unread_count = unread_count(b);
+    if (a_unread_count < b_unread_count) {
+        return 1;
```

Comment:
Currently, I've noticed that the icon position for ascending/descending sort is incorrect. When you sort from a high to low number of unreads, the icon seems to be in the opposite state (if I'm understanding it correctly).

![image](https://github.com/zulip/zulip/assets/53193850/9cd900f5-c84b-4897-9ec5-f598377719aa)

Reply:
Fixed